packages/frontend-2/components/project/page/team/Block.vue

```diff
@@ -3,50 +3,44 @@
     <template #top>
       <div class="flex items-center">
         <div class="flex items-center justify-between w-full">
-          <div
-            v-if="activeUser"
-            class="flex items-center gap-0.5 flex-grow select-none"
-          >
-            <Cog6ToothIcon class="h-5 w-5" />
-            <span class="text-sm">Settings</span>
-          </div>
-          <div v-else class="flex items-center gap-0.5 flex-grow select-none">
+          <div class="flex items-center gap-1 flex-grow select-none">
             <UsersIcon class="h-5 w-5" />
-            <span class="text-sm">Team</span>
+            <span class="text-sm font-bold">Team</span>
           </div>
-          <div class="flex items-center text-xs">
+          <div class="flex items-center text-sm">
             {{ project.role?.split(':').reverse()[0] }}
           </div>
         </div>
       </div>
     </template>
     <template #bottom>
-      <div class="flex items-center justify-between mt-3">
+      <div class="flex items-center justify-between mt-1">
         <UserAvatarGroup :users="teamUsers" class="max-w-[104px]" />
         <div v-if="activeUser">
-          <FormButton class="ml-2" @click="onButtonClick">
-            {{ project.role === 'stream:owner' ? 'Manage' : 'View' }}
+          <FormButton class="ml-2" :icon-left="UserPlusIcon" @click="onButtonClick">
+            Invite
           </FormButton>
         </div>
       </div>
     </template>
     <template #default>
-      <ProjectPageTeamDialog
+      <ProjectPageInviteDialog
         v-model:open="dialogOpen"
-        :project="project"
-        :open-section="openSection"
+        :disabled="!isOwner"
+        :disabled-message="
+          !isOwner ? 'Sorry, you must be a project owner to invite users' : undefined
```

Comment:
Misleading API - the disabledMessage string is never actually used. It's just used like a boolean